stripe/stripe_object.py

```diff
@@ -417,3 +434,12 @@ def __deepcopy__(self, memo):
             super(StripeObject, copied).__setitem__(k, deepcopy(v, memo))
 
         return copied
+
+    _inner_class_types: ClassVar[Dict[str, Type["StripeObject"]]] = {}
+    _inner_class_dicts: ClassVar[List[str]] = []
+
+    def _get_inner_class_type(self, field_name: str):
+        return self._inner_class_types.get(field_name)
```

Comment:
Won't this fail if _inner_class_types is None?

Reply:
It won't be, `inner_class_types` defaults (on the parent class) to an empty dict.